misc/upgrades-installed-check

```diff
@@ -23,6 +23,13 @@ elif [ -e /etc/debian_version ]; then
    apt_get_upgrade_output="$(LANG="C" apt-get -s upgrade 2>&1)"
    exit_code="$?"
    echo "$apt_get_upgrade_output" | awk "/^Inst/{ print $2 }" | [ "$(wc -L)" -eq 0 ] && echo "true" || echo "false"
+elif [ -e /etc/arch-release ]; then
+    ## Archlinux
+    set -e
+    set -o pipefail
+    checkupdates_output="$(checkupdates 2>&1)"
+    exit_code="$?"
+    echo "$checkupdates_output" | { grep -F -- '->' || true; } | [ "$(wc -L)" -eq "0" ] && echo "true" || echo "false"
```

Comment:
wc seems useless here, grep itself can set appropriate exit code:
```
! echo "$checkupdates_output" | grep -qF -- '->' && echo "true" || echo "false"
```
(note `!` for negation)

Reply:
Do we need the negation? I like using grep's exit code directly, it greatly simplifies the expression, but can't we just switch true and false like this?
```
echo "$checkupdates_output" | grep -qF -- '->' && echo "false" || echo "true"
```
If there are updates, grep will match the arrows, exit with 0 and will output "false" (i.e. "there are pending upgrades"), otherwise it will output true ("all upgrades have been installed").

Reply:
Yes, that's even better.